extreme_classification/extreme_ml.ipynb

```diff
@@ -33,7 +33,7 @@
   },
```

Comment:
<p>Just a change in text.</p><p>Can you make 'Turn off the service once you are sure that you do not want to use it anymore. Once the service is stopped, you cannot use it again' to 'Delete the index once you do not want to use it anymore. Once the index is deleted, you cannot use it again'</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>